### DISCUSSION TITLE
Mulig å inkludere gangtid til stopp i tidslinje

### DISCUSSION THREAD

**Discussion by yngvar** [entur/tavla#310] [Ideas]
Jeg har lagt til et par forskjellige stopp i min oversikt. 
Stopp Sandstuveien ligger ca. 4 minutts gange fra adressen jeg har oppgitt som utgangspunkt. 
Stopp Ryen ligger c. 10 minutts gange. 
Det hadde vært fantastisk å kunne velge at omtrentlig gangavstand i tid kan legges til tidslinja på et eller annet vis, slik at jeg enda bedre kan skjønne når jeg må forlate utgangspunktet.

Når jeg skal reise så er det den faktiske beregningen jeg gjør, bør jeg gå til Ryen eller til Sandstuveien for å rekke neste avgang?

![image](https://user-images.githubusercontent.com/50703/79717285-7de69900-82d9-11ea-82fc-9c61ba1d444a.png)

**Comment by draperunner** [comment]
Hei! Dette er eit veldig godt forslag. Takk! Me har faktisk tenkt på dette og har det i idélista vår. Så me kjem nok til å løyse dette, men eg tør ikkje seie noko om når. det skjer.

For å fylle på litt info på issuet:

Alle tre visningstypane bør ha denne typen informasjon. Dette må me skisse litt på.

Er det tilstrekkelig med luftlinjeestimering av gangtid, eller skal me gjere `foot`-søk med SDKen? Sistnevnte er det mest nøyaktige, men kan også gi rare tider i tilfelle ikkje alle stiar/vegar er kobla opp riktig.

**Comment by yngvar** [comment]
> Er det tilstrekkelig med luftlinjeestimering av gangtid, eller skal me gjere `foot`-søk med SDKen? Sistnevnte er det mest nøyaktige, men kan også gi rare tider i tilfelle ikkje alle stiar/vegar er kobla opp riktig.

Jeg legger neppe opp flere enn et par avgangssteder per tavle, så for meg hadde det vært vel så greit å kunne oppgi det når jeg registrerer holdeplassen, altså manuelt kunne skrive antall minutter gangvei/transport  fra utgangspunkt til holdeplass, per holdeplass jeg registrerer. Dette vet jeg godt, og sannsynligvis mye mer presist enn algoritmer klarer beregne. Jeg et at jeg bruker fire minutter i min lett bedagelige gangfart til Sandstuveien og 10-11 minutter til t-banen på Ryen.
Det kompliserer sikkert deres modell, minuttangivelse istedet for et boolsk "ta-med-gangtid"-attributt, men det hadde gitt meg større verdi.

**Comment by forteller** [comment]
Dette er interessant. Da kunne man jo, isteden eller i tillegg, til å nevne gangavstanden for hver avgang, så kunne man hatt en boks/widget som til enhver tid oppdaterer seg med hvilken avgang som er den neste du kan rekke i en bestemt retning (mot sentrum, fra sentrum). 

Denne boksen kunne inneholde både navnet på linjen og holdeplassen i tillegg til klokkeslettet når banen/bussen går fra stasjonen, og når du må gå ut døra for å rekke den.

Litt sånn her:

Neste avgang mot sentrum er:
XX [linjenavn] fra [holdeplass] kl YY:ZZ
Gå senest innen Æ minutter for å rekke den.

**Comment by draperunner** [comment]
Hei! No har me lagt til gangtid i både tekslig og visuell form på Tidslinja:

<img width="483" alt="image" src="https://user-images.githubusercontent.com/4339443/126795673-74d6aa95-e166-4c81-8420-73290b86dfad.png">

Gangtida er basert på eit _gangreisesøk_. Altså ikkje estimering basert på luftlinje, men den faktisk vegen du må gå er tatt hensyn til.

**Reply by forteller** [reply]
Supert! En ting jeg alltid har lurt på når det gjelder dette: Tar utregningene dere bruker hensyn til at det ofte tar lenger tid å gå om vinteren, fordi det enten er snø eller glatt, og at det tar spesielt lang tid å gå glatte bakker?

**Reply by draperunner** [reply]
Nei, det blir antatt at du går i konstant fart uansett føre, forhold, oppover- og nedoverbakker. Her er det ca. 1,3 m/s (ca. 4,7 km/t) som er brukt som gangfart. Og me runder tida opp til første heile minutt.

**Reply by forteller** [reply]
Det var det jeg trodde/fryktet. Er det noe dere har planer om å endre? Det kan være veldig stor forskjell på hvor lang tid det tar å gå utfra føreforhold.

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
